lib/ui/README.md

```diff
@@ -14,7 +14,6 @@ You can configure it by providing a provider API.
 -   [Hacking Guide](#hacking-guide)
```

Comment:
80% of this 'guide' is outdated.. It's also pretty much never been useful to anyone I think.

I don't want to sink too much time into making this document correct..